xml/soc-operations-gpu-passthrough.xml

```diff
@@ -128,18 +114,170 @@ drwxr-xr-x 3 root root 0 Feb 19 16:09 /sys/kernel/iommu_groups/20/devices/0000:0
+    </para>
+    </step>
+    <step>
+    <para>Once the file is updated, restart nova api on the controller node</para>
+<screen>&prompt.root;systemctl restart openstack-nova-api
+</screen>
+    </step>
+    </procedure>
+    </listitem>
+
+    <listitem><para>
+    <emphasis role="bold">Flavor Creation</emphasis>
+    </para>
+    <para>
+    Login into the controller node and create a new flavor or update an existing flavor with the property
+    pci_passthrough:alias. Example:
+    </para>
+<screen>&prompt.root;source .openrc
+openstack flavor create --ram 8192 --disk 100 --vcpu 8 gpuflavor
+openstack flavor set gpuflavor --property "pci_passthrough:alias"="a1:1"
+</screen>
+    <para>
+    In the property, "pci_passthrough:alias"="a1:1", a1 before the : references the alias name as provided in the
+    configuration entries while the number 1 after the : tells Nova that a single GPU should be assigned.
+    </para>
+    </listitem>
+    <listitem>
+    <para>
+    <emphasis role="bold">Boot an instance with the flavor created in previous step</emphasis>
+    </para>
+    <para>
+      Make sure the VM becomes ACTIVE. Login into the virtual instance and verify that GPU is seen from the guest
+      by running the lspci command on the guest.
+    </para>
+    </listitem>
+    </orderedlist>
+    </listitem>
   </orderedlist>
+   <para>Few additional <emphasis role="bold"> examples</emphasis></para>
+   <orderedlist>
+     <listitem>
+       <para>Example 1: Multiple compute hosts</para>
+       <para>Compute host x:</para>
+<screen>
+[pci]
+passthrough_whitelist = [{"address": "0000:84:00.0"}]
+alias = {"vendor_id": "10de", "name": "a1", "device_type": "type-PCI", "product_id": "1db4"}
+</screen>
+       <para>Compute host y:</para>
+<screen>
+[pci]
+passthrough_whitelist = [{"address": "0000:85:00.0"}]
+alias = {"vendor_id": "10de", "name": "a1", "device_type": "type-PCI", "product_id": "1db4"}
+</screen>
+       <para>Controller nodes:</para>
+<screen>
+[pci]
+alias = {"vendor_id": "10de", "name": "a1", "device_type": "type-PCI", "product_id": "1db4"}
+</screen>
+
+      </listitem>
+       <listitem>
+       <para>Example 2: Multiple PCI devices on the same host </para>
+         <para>Compute host z:</para>
+<screen>
+[pci]
+passthrough_whitelist = [{"vendor_id": "10de", "product_id": "1db4"}, {"vendor_id": "10de", "product_id":
+"1db1"}]
+alias = {"vendor_id": "10de", "name": "a2", "device_type": "type-PCI", "product_id": "1db1"}
+alias = {"vendor_id": "10de", "name": "a1", "device_type": "type-PCI", "product_id": "1db4"}
+</screen>
+         <para>Controller nodes:</para>
+<screen>
+[pci]
+alias = {"vendor_id": "10de", "name": "a2", "device_type": "type-PCI", "product_id": "1db1"}
+alias = {"vendor_id": "10de", "name": "a1", "device_type": "type-PCI", "product_id": "1db4"}
+</screen>
+         <para> In order to pass both the devices to the instance, you can set the "pci_passthrough:alias"="a1:1,a2:1"
+         </para>
```

Comment:
it might be worth a screen section showing the multiple pci passthrough setting (presumably this is something like `openstack flavor set gpuflavor --property "pci_passthrough:alias"="a1:1,a2:1"`  ) ... I think that would wrap things up a little better, as it is the line here is useful, but feels like there's supposed to be something after it to close out the section.

Reply:
Done